src/vizier/services/adaptive_export/internal/clickhouse/insert_test.go

```
@@ -0,0 +1,110 @@
+}
+
+// TestInsertSQL_Unknown — defensive contract.
+func TestInsertSQL_Unknown(t *testing.T) {
+	for _, bad := range []string{"", "evil; DROP TABLE"} {
+		_, err := InsertSQL(bad)
+		if !errors.Is(err, ErrUnknownTable) {
+			t.Fatalf("InsertSQL(%q) → %v, want ErrUnknownTable", bad, err)
+		}
+	}
+}
+
```

golangci-lint (2.12.1)

[error] File is not properly formatted

(gci)